LoggableBehavior.php

```diff
@@ -1,166 +1,174 @@
-		if(count(array_diff_assoc($newattributes, $oldattributes)) <= 0) return;
-
-		// If this is a new record lets add a CREATE notification
-		if ($insert){
-			$this->leaveTrail('CREATE');
-		}
-
-		// Now lets actually write the attributes
-		$this->auditAttributes($insert, $newattributes, $oldattributes);
-		
-		// Reset old attributes to handle the case with the same model instance updated multiple times
-		$this->setOldAttributes($this->owner->getAttributes());
-	}
-
-	public function auditAttributes($insert, $newattributes, $oldattributes = array())
-	{
-		foreach ($newattributes as $name => $value) {
-			$old = isset($oldattributes[$name]) ? $oldattributes[$name] : '';
-
-			// If we are skipping nulls then lets see if both sides are null
-			if($this->skipNulls && empty($old) && empty($value)){
-				continue;
-			}
-
-			// If they are not the same lets write an audit log
-			if ($value != $old) {
-				$this->leaveTrail($insert ? 'SET' : 'CHANGE', $name, $value, $old);
-			}
-		}
-	}
-
-	public function leaveTrail($action, $name = null, $value = null, $old_value = null)
-	{
-		$log			= new AuditTrail();
-		$log->old_value = $old_value;
-		$log->new_value = $value;
-		$log->action	= $action;
-		$log->model		= $this->owner->className(); // Gets a plain text version of the model name
-		$log->model_id	= (string)$this->getNormalizedPk();
-		$log->field		= $name;
-		$log->stamp		= $this->storeTimestamp ? time() : date($this->dateFormat); // If we are storing a timestamp lets get one else lets get the date
-		$log->user_id	= (string)$this->getUserId(); // Lets get the user id
-		return $log->save();
-	}
-	
-	public function getOldAttributes()
+	}
+
+	public function auditAttributes($insert, $newattributes, $oldattributes = array())
+	{
+		foreach ($newattributes as $name => $value) {
+			$old = isset($oldattributes[$name]) ? $oldattributes[$name] : '';
+
+			// If we are skipping nulls then lets see if both sides are null
+			if($this->skipNulls && empty($old) && empty($value)){
+				continue;
+			}
+
+			// If they are not the same lets write an audit log
+			if ($value != $old) {
+				$this->leaveTrail($insert ? 'SET' : 'CHANGE', $name, $value, $old);
+			}
+		}
+	}
+
+	public function leaveTrail($action, $name = null, $value = null, $old_value = null)
+	{
+		if ($this->active) {
+			$log			= new AuditTrail();
+			$log->old_value = $old_value;
+			$log->new_value = $value;
+			$log->action	= $action;
+			$log->model		= $this->owner->className(); // Gets a plain text version of the model name
+			$log->model_id	= (string)$this->getNormalizedPk();
+			$log->field		= $name;
+			$log->stamp		= $this->storeTimestamp ? time() : date($this->dateFormat); // If we are storing a timestamp lets get one else lets get the date
+			$log->user_id	= (string)$this->getUserId(); // Lets get the user id
+			return $log->save();
+		}

```

Comment:
Indeed, basically delete the space and new line on line 139 so that the code goes from:

    }
    else{

to:

    }else{